install_config/aggregate_logging.adoc

```diff
@@ -750,32 +747,23 @@ As part of the OpenShift cluster installation, it is recommended that you add th
 Fluentd node selector to the list of persisted
 xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[node labels].
 
-[[fluentd-use-journald]]
-*Having Fluentd Use the Systemd Journal as the Log Source*
+Fluentd uses `journald` as the system log source. These are log
+messages from the operating system, from Docker, and from OpenShift. For
+container logs, Fluentd will determine which log driver Docker is using,
+`json-file` or `journald`, and will automatically read the logs from that source.
 
-By default, Fluentd reads from *_/var/log/messages_* and
-*_/var/log/containers/<container>.log_* for system logs and container logs,
-respectively. You can instead use the systemd journal as the log source. There
-are three inventory parameters available:
+// [[fluentd-use-journald]]
+// *Having Fluentd Use the Systemd Journal as the Log Source*
```

Comment:
Does the leading `//` mean this section is commented out?

Reply:
It does. I wasn't certain that content was still applicable, but didn't want to delete it just in case.